webapp/src/components/AuctionSplash/AuctionSplash.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Input, Form } from 'semantic-ui-react'
+import { t } from '@dapps/modules/translation/utils'
+
+import AuctionStaticPage from 'components/AuctionStaticPage'
+import AuctionCountdown from 'components/AuctionCountdown'
+
+export default class AuctionFinishedPage extends React.PureComponent {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired
+  }
+
+  render() {
+    const { onSubmit } = this.props
+
+    return (
+      <AuctionStaticPage>
+        <div className="AuctionSpash">
+          <AuctionCountdown>
+            <p>{t('auction_splash.cta_title')}</p>
+            <Form onSubmit={onSubmit} className="cta-wrapper">
```

Comment:
I think need to add preventDefault from `lib/utils` here?
```javascript
import { preventDefault } from 'lib/utils'
```

```suggestion
            <Form onSubmit={preventDefault(onSubmit)} className="cta-wrapper">
```